aries-site/src/layouts/main/Footer.js

```diff
@@ -30,29 +36,36 @@ export const Footer = () => {
         horizontal: size !== 'small' ? 'xlarge' : 'large',
       }}
     >
-      <Box direction="row-responsive">
-        <ThemeModeToggle />
-      </Box>
       <Box direction="row-responsive" gap="medium" align="center">
```

Comment:
I think we should decrease the gap to match what is in the Header because it feels like there is too much space now between buttons since they have their own pad also.